### PR TITLE
Strengthen Bot Check

### PR DESCRIPTION
**Pull request by IThinkImOKAY**
## Description
TLDR: Replace the first character in the `X-User-Type` parameter with an upper-case version of the character.

## Motivation and Context
I was sending a few API requests with the `X-User-Type` parameter set to `bot`. Realized the `is_bot` attribute was missing.

## How Has This Been Tested?
There isn't much to test.

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by IThinkImOKAY** [comment]
@captainmeta4 updated

**Review by captainmeta4 — APPROVED** [review]
